src/data/members.ts

```diff
@@ -1,65 +1,352 @@
-import { ResearchTopicType } from './publications'
+export const CareerTypes = [
```

Comment:
are you using this constant anywhere in the code? like `CareerTypes.map()`? Otherwise we can define the types only.
export type CareerType = 'Associate Professor'
|  'Postdoc Researcher',
|  'Ph.D. Student',
|   'M.S. Student',
...

Reply:
Yeah, I think Noza is using this to build the navigation menu on the side. Could we do that just using `type`?